src/Common/tests/System/Net/Http/Http2LoopbackServer.cs

```diff
@@ -619,7 +627,7 @@ public async Task SendResponseBodyAsync(int streamId, ReadOnlyMemory<byte> respo
                 throw new Exception("Response body too long");
             }
 
-            await SendResponseDataAsync(streamId, responseBody, true);
+            await SendResponseDataAsync(streamId, responseBody, true).ConfigureAwait(false);
```

Comment:
Why are we adding all these '.ConfigureAwait(false)' to test code?  I remember a discussion with @stephentoub  that we shouldn't be doing that.  Unlike product sources, the test code should be running in an environment where adding .ConfigureAwait is not only unnecessary but also can mask real problems in the product code.

I'll see if I can find that past discussion with @stephentoub

Reply:
Thx for the forwarding the old email:

Excerpt for reference:

```
From: Stephen Toub
Date: Tuesday, February 27, 2018 at 6:45 PM
Subject: RE: Async question

Re: “LoopbackServer uses ConfigureAwait a bunch”
 Ah, I should clarify my statement.  I make a mental distinction between app code and library code.  And my default rule is app code doesn’t use ConfigureAwait (because 99% of the time it should go back to the original sync ctx) and library code always uses ConfigureAwait (because 99% of the time it doesn’t need to go back to the original sync ctx, and doing so can incur perf costs and cause issues like deadlocks).  I view tests as being in the app code category.  I view LoopbackServer as being in the library code category.

 Re: “One thing I don’t understand is whether XUnit has its own SynchronizationContext (I think it does), and if so, why/what it does.  Stephen do you understand this?”
 It does.  It uses a custom SynchronizationContext to a) manage concurrency in the tests and how many are allowed to execute at any one time, and b) to enable async void tests.
```

So, ok, these `ConfigureAwait(false)` are ok for Loopback server code.